build.gradle

```diff
@@ -70,3 +71,17 @@ tasks.named('test') {
 bootRun {
   systemProperties = System.properties
 }
+
+spotless {
+  java {
+    target fileTree('.') {
+        include '**/*.java'
+        exclude '**/build/**', '**/build-*/**'
+    }
+    toggleOffOn()
+    palantirJavaFormat()
+    removeUnusedImports()
+    trimTrailingWhitespace()
+    endWithNewline()
+  }
+}
```

Comment:
Format rules and configuration look good. Please consider adding Spotless rule execution so that you can run formatter automatically before pushing code to the repository.

`test.dependsOn("spotlessApply")`

Reply:
Instead of doing this command:
`gradle -Dspring.config.location=$HOME/.vetlog/application-development.yml spotlessApply`

You can do just:
`gradle -Dspring.config.location=$HOME/.vetlog/application-development.yml test`